src/Definition.php

```diff
@@ -102,23 +89,17 @@ public function addMethods(array $methods)
     /**
      * Set all methods at once (overwrite)
      *
-     * @param  array $methods Array of \Laminas\Server\Method\Definition objects or arrays
-     * @return \Laminas\Server\Definition
+     * @param  Method\Definition[]
```

Comment:
missing param name - should be `$methods`

Reply:
Fixed.